### DISCUSSION TITLE
es/models/yolo11/

### DISCUSSION THREAD

**Discussion by giscus** [ultralytics/ultralytics#18179] [Docs]
# es/models/yolo11/

Descubra YOLO11, el último avance en detección de objetos de última generación, que ofrece una precisión y eficacia inigualables para diversas tareas de visión por ordenador.

https://docs.ultralytics.com/es/models/yolo11/

**Comment by UltralyticsAssistant** [comment]
👋 Hola, ¡gracias por tu interés en Ultralytics 🚀! Te animamos a visitar nuestra [Documentación](https://docs.ultralytics.com/es/models/yolo11/) para conocer más detalles sobre YOLO11, el último avance en detección de objetos, y explorar cómo puedes empezar a utilizarlo.

Si esto es un 🐛 Reporte de error, por favor proporciona un [ejemplo reproducible mínimo (ERM)](https://docs.ultralytics.com/help/minimum_reproducible_example/) que pueda ayudarnos a diagnosticar el problema.

Si tienes una pregunta sobre entrenamiento personalizado ❓, incluye tanta información como sea posible, como ejemplos de imágenes del dataset y registros de entrenamiento, y asegúrate de seguir nuestros [Consejos para obtener los mejores resultados de entrenamiento](https://docs.ultralytics.com/guides/model-training-tips/).

Únete a la comunidad de Ultralytics en el espacio que prefieras para colaborar y aprender más: 
- Para chat en tiempo real, únete al [Discord](https://discord.com/invite/ultralytics) 🎧. 
- Para discusiones más extensas, visita nuestro [Discourse](https://community.ultralytics.com).
- O revisa y contribuye en el [Subreddit](https://reddit.com/r/Ultralytics) oficial.

## Actualización

Asegúrate de que estás utilizando siempre la última versión del paquete `ultralytics` junto con todos los [requisitos](https://github.com/ultralytics/ultralytics/blob/main/pyproject.toml) en un entorno con [**Python>=3.8**](https://www.python.org/) y [**PyTorch>=1.8**](https://pytorch.org/get-started/locally/):

```bash
pip install -U ultralytics
```

## Entornos

YOLO puede ejecutarse en los siguientes entornos verificados y actualizados (con todas las dependencias necesarias como [CUDA](https://developer.nvidia.com/cuda)/[CUDNN](https://developer.nvidia.com/cudnn), [Python](https://www.python.org/) y [PyTorch](https://pytorch.org/) preinstaladas):

- **Notebooks** con GPU gratis: <a href="https://console.paperspace.com/github/ultralytics/ultralytics"><img src="https://assets.paperspace.io/img/gradient-badge.svg" alt="Run on Gradient"/></a> <a href="https://colab.research.google.com/github/ultralytics/ultralytics/blob/main/examples/tutorial.ipynb"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"></a> <a href="https://www.kaggle.com/models/ultralytics/yolo11"><img src="https://kaggle.com/static/images/open-in-kaggle.svg" alt="Open In Kaggle"></a>
- **Google Cloud** VM de aprendizaje profundo. Consulta la [Guía rápida de GCP](https://docs.ultralytics.com/yolov5/environments/google_cloud_quickstart_tutorial/)
- **Amazon** AMI de aprendizaje profundo. Consulta la [Guía rápida de AWS](https://docs.ultralytics.com/yolov5/environments/aws_quickstart_tutorial/)
- **Docker Image**. Consulta la [Guía rápida de Docker](https://docs.ultralytics.com/yolov5/environments/docker_image_quickstart_tutorial/) <a href="https://hub.docker.com/r/ultralytics/ultralytics"><img src="https://img.shields.io/docker/pulls/ultralytics/ultralytics?logo=docker" alt="Docker Pulls"></a>

## Estado

<a href="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule"><img src="https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml/badge.svg" alt="Ultralytics CI"></a>

Si este badge está en verde, todas las pruebas de [Ultralytics CI](https://github.com/ultralytics/ultralytics/actions/workflows/ci.yaml?query=event%3Aschedule) están pasándose correctamente. Estas pruebas verifican la operación correcta de todos los [Modos](https://docs.ultralytics.com/modes/) y [Tareas](https://docs.ultralytics.com/tasks/) de YOLO en macOS, Windows y Ubuntu diariamente y en cada commit.

Esta es una respuesta automatizada: un ingeniero de Ultralytics también te ayudará pronto 😊.
